platform-dom/src/main/java/com/softicar/platform/dom/elements/input/auto/DomAutoCompleteDisplayStringDeduplicator.java

```diff
@@ -32,19 +34,23 @@
 
 	private final Function<T, IDisplayString> displayFunction;
 	private final Comparator<T> comparator;
```

Comment:
maybe rename `comparator` to `valueComparator` and put it next to `keyComparator`?

Reply:
ok